### PR TITLE
Handle interface type

### PR DESCRIPTION
**Pull request by abiewardani**
Handle Interface type

### CODE REVIEW

**Comment by antonius-hilman** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://build.bhinnekalocal.com:8080//job/goson-PR/5/

File: json_unmarshal.go [hunk 1]
```diff
@@ -51,6 +51,9 @@ func scanTarget(target reflect.Value, source interface{}) {
 	case reflect.Ptr:
 		scanTarget(target.Elem(), source)
 
+	case reflect.Interface:
```
Comment (agungdwiprasetyo) [inline json_unmarshal.go]: dipindah ke function `setValue()`

**Review by agungdwiprasetyo — CHANGES_REQUESTED** [review]

**Comment by antonius-hilman** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://build.bhinnekalocal.com:8080//job/goson-PR/6/

**Review by agungdwiprasetyo — APPROVED** [review]
mantul

**Pull request merged**